content/en/docs/started/support.md

```diff
@@ -4,224 +4,101 @@ description = "Where to get support for Kubeflow"
-  <table class="table table-bordered">
-    <thead class="thead-light">
-      <tr>
-        <th>Organization</th>
-        <th>Points of contact</th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr>
-        <td><a href="https://www.arrikto.com">Arrikto</a></td>
-        <td><a href="mailto:sales@arrikto.com">sales@arrikto.com</a></td>
-      </tr>
-      <tr>
-        <td><a href="https://www.ubuntu.com">Canonical</a></td>
-        <td><a href="https://ubuntu.com/kubeflow#get-in-touch">Get in touch</a></td>
-      </tr>
-      <tr>
-        <td><a href="https://www.pattersonconsultingtn.com/">Patterson Consulting</a></td>
-        <td> 
-        <a href="http://www.pattersonconsultingtn.com/offerings/managed_kubeflow.html">Managed Kubeflow</a></td>
-      </tr>
-      <tr>
-        <td><a href="https://www.seldon.io/">Seldon</a></td>
-        <td> 
-        <a href="https://github.com/SeldonIO/seldon-core/issues">Issue 
-        tracker</a></td>
-      </tr>    
-    </tbody>
-  </table>
-</div>
-
-<a id="cloud-support"></a>
-## Support from a cloud or platform provider
-
-If you're using the services of a cloud provider to host Kubeflow, the cloud
-provider may be able to help you diagnose and solve a problem.
-
-Consult the support page for the cloud service or platform that you're using:
+## Support from commercial providers in the Kubeflow Ecosystem
 
-* [Amazon Web Services (AWS)](https://aws.amazon.com/contact-us/)
-* [Canonical Ubuntu](https://ubuntu.com/kubeflow#get-in-touch)
-* [Google Cloud Platform (GCP)](https://cloud.google.com/support-hub/)
-* [IBM Cloud](https://www.ibm.com/cloud/support)
-* [Microsoft Azure](https://azure.microsoft.com/en-au/support/options/)
-* [Red Hat OpenShift](https://help.openshift.com/)
+We want to promote commercial companies and idividuals that contribute back to the open source project.
+Below is a table of organizations that contribute to Kubeflow and offer commercial support:
 
-## Other places to ask questions
+| Provider                      | Support Link                                                                                                                                                    |
+|-------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aranui Solutions              | [Aranui Solutions](https://www.aranui.solutions/services)                                                                                                       |
+| Canonical                     | [Ubuntu Kubeflow](https://ubuntu.com/kubeflow#get-in-touch)                                                                                                     |
+| Freelancer Julius von Kohout  | [LinkedIn](https://de.linkedin.com/in/juliusvonkohout/), [Slack](https://cloud-native.slack.com/team/U06LW431SJF), [GitHub](https://github.com/juliusvonkohout) |
```

Comment:
I want to be explicit that I am pro listing individual free lancers here as they should be rewarding for contributing to the community. 

We can vet this via the KSC for now but we probably will want some process of users showing contributions. 

I am very happy to celebrate commercial interests benefiting from their contributions to Kubeflow.

Reply:
@franciscojavierarceo  should we change the title then based on this?

Right now is "Support from commercial providers in the Kubeflow Ecosystem"

maybe something like "Support from commercial providers and individual in the Kubeflow Ecosystem"

Reply:
How is an individual not a commercial providers if he provides commercial services? But that would anyway be something for follow up PRs.

Reply:
Yeah +1, I suspect that the person operates under a commercial entity.